types/gltf/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
When adding new definitions you cannot have any rules here. Only extends clause.

Reply:
Hm I see. I take that to mean that `IFoo` naming is disallowed on all typings in this repository?

@bghgary are you OK with the renaming this would involve for BabylonJS?

Reply:
This is the naming convention we are using in BabylonJS for many interfaces. Types can be aliased when imported (especially with modules), so hopefully it will work out.

Reply:
I'll remove the `I` prefixes then, but let me know if there is any issue.

Reply:
Done ✅